docs/api/supersuit_wrappers.md

```diff
@@ -77,103 +77,172 @@ Supersuit includes the following wrappers:
 
 * `concat_vec_envs_v0(vec_env, num_vec_envs, num_cpus=0, base_class='gym')` takes in an `vec_env` which is vector environment (should not have multithreading enabled). Creates a new vector environment with `num_vec_envs` copies of that vector environment concatenated together and runs them on `num_cpus` cpus as balanced as possible between cpus. `num_cpus=0` or `num_cpus=1` means to create 0 new threads, i.e. run the process in an efficient single threaded manner. A use case for this function is given below. If the base class of the resulting vector environment matters as it does for stable baselines, you can use the `base_class` parameter to switch between `"gym"` base class and `"stable_baselines3"`'s base class. Note that both have identical functionality.
 
-### Parallel Environment Vectorization
+[//]: # (### Parallel Environment Vectorization)
```

Comment:
This is new to me, what does this do?

Reply:
Just commenting out markdown (done via pycharm)

Yeah I/we need to review the wrappers part